hbase-http/src/main/java/org/apache/hadoop/hbase/http/prometheus/PrometheusHadoopServlet.java

```diff
@@ -35,13 +35,18 @@
 
 @InterfaceAudience.Private
 public class PrometheusHadoopServlet extends HttpServlet {
+  private boolean descriptionEnabled;
+  private String queryParam;
```

Comment:
Servlets should be stateless. There is only one `PrometheusHadoopServlet` in the JVM, so it can lead to concurrency issues when multiple requests are coming in the same time.

Reply:
Should've paid more attention to it, thank you for your suggestion! 
I fixed it in the next commit.